x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Analyzer.java

```diff
@@ -403,6 +409,12 @@ else if (!fa.dataType().isPrimitive()) {
                 // since this is an unresolved start we don't know whether it's a path or an actual qualifier
                 Attribute q = resolveAgainstList(us.qualifier(), output);
 
+                // the wildcard coulnd't be expanded because the field doesn't exist at all
```

Comment:
typo: couldn't
the two comments don't make sense "the wildcard couldn't .. at all to the list of expanded)?